src/xunit.performance.metrics/InstructionsRetiredEvaluator.cs

```diff
@@ -30,6 +30,7 @@ internal void Initialize(PerformanceMetricEvaluationContext context, int profile
                     }
                     else
                     {
+                        // FIXME: We should fail here instead.
```

Comment:
Agreed.

Reply:
This has been the behavior from day 1. I'll postpone this change to future PR.

---
In reply to: [143036663](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143036663) [](ancestors = 143036663)